ksqldb-engine/src/test/java/io/confluent/ksql/services/SandboxedSchemaRegistryClientTest.java

```diff
@@ -64,6 +64,7 @@ public static Collection<TestCase<SchemaRegistryClient>> getMethodsToTest() {
           .ignore("register", String.class, Schema.class, int.class, int.class)
           .ignore("register", String.class, ParsedSchema.class, int.class, int.class)
           .ignore("getLatestSchemaMetadata", String.class)
+          .ignore("registerWithResponse", String.class, ParsedSchema.class, boolean.class)
```

Comment:
Why is this added here? This list is for methods which are _not_ implemented by the proxy.

Reply:
It was failing before. Looks like this is a list of method implemented by proxy which will be ignored and not throw unsupported error

Reply:
This time, it's complaining about
```
  default String tenant() {
    return DEFAULT_TENANT;
  }
```

Reply:
I see. Turns out we have to add the new method in both places, since our sandbox client provides a custom implementation. Should be fixed now in the latest commit.